### PR TITLE
Add installation instructions from cargo

### PR DESCRIPTION
**Pull request by zhmushan**

### CODE REVIEW

**Comment by Schniz** [comment]
maybe we need to publish it to cargo as well. It's just a `cargo publish` away

**Comment by zhmushan** [comment]
That's cool!  In fact, the first command I tried was ` cargo install fnm `, but it didn’t work

File: README.md [hunk 1]
```diff
@@ -101,6 +101,12 @@ Then apply the changes the installer prints, to set up your shell profile.
     FOR /f "tokens=*" %i IN ('fnm env --use-on-cd') DO CALL %i
     ```
 
+#### Using Cargo (Linux/Mac/Windows)
+
+```bash
+cargo install --git https://github.com/Schniz/fnm.git
```
Comment (Schniz) [inline README.md:107]:
```suggestion
cargo install fnm
```

Comment (Schniz) [reply README.md:107]: I published the last version 😸

**Comment by Schniz** [comment]
Thanks @zhmushan! 😁

**Pull request merged**